board/netconf/rootfs/libexec/infix/yanger

```diff
@@ -338,6 +338,42 @@ def add_ospf(ospf):
     insert(ospf, "ietf-ospf:areas", "area", areas)
     add_ospf_routes(ospf)
 
+def get_bridge_port_pvid(ifname, iface_out, test):
+    if test:
+        cmd = ['cat', f"{test}/bridge-vlan-show-dev-{ifname}.json"]
+    else:
+        cmd = ['bridge', '-j', 'vlan', 'show', 'dev', ifname]
+
+    data = run_json_cmd(cmd)
```

Comment:
This is not specific to this PR, just something that popped into my head:

Suppose we kept `test` as a global variable, and then had a single check in `run_json_cmd()` that would  translate the argument vector into a filename (e.g., to something like `{test}/bridge--j-vlan-show-dev-{ifname}.json` in this case) if `test` is set.

Reply:
I agree. The reason for this initial implementation is that I was afraid we would end up with commands such as `foo --bar="hello"` which converted to a filename would not look pretty.

I can fix that in another PR. There's more cosmetic inconsistencies that I would like to address.